### ISSUE TITLE
Should the consumer install `babel` explicitly?

### ISSUE THREAD

**Issue by sarbbottam** [stylelint/stylelint#293]
Please find the content of the `gulpfile.js` as below mentioned.
```js
var gulp = require('gulp');
var postcss = require('gulp-postcss');
var stylelintConfig = require('stylelint-config-suitcss');

gulp.task('postcss', function () {
  var processors = [
    require('stylelint')(stylelintConfig),
    require('postcss-reporter')()
  ];
  return gulp.src('./src/css/main.css')
    .pipe(postcss(processors))
    .pipe(gulp.dest('./dist/css'));
});
```
It errors out for the usage of `Set`.

```zsh
吽 node --version
v0.10.40
[zsh] sarbbo at rushpress-lm in ~/GitHub/sheer (master)
吽 gulp
[09:41:26] Using gulpfile ~/GitHub/sheer/gulpfile.js
[09:41:26] Starting 'postcss'...
[09:41:26] 'postcss' errored after 217 ms
[09:41:26] ReferenceError: Set is not defined
    at Object.<anonymous> (/Users/sarbbo/GitHub/sheer/node_modules/stylelint/dist/rules/number-zero-length-no-unit/index.js:21:23)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/sarbbo/GitHub/sheer/node_modules/stylelint/dist/rules/index.js:197:31)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
[zsh] sarbbo at rushpress-lm in ~/GitHub/sheer (master)
吽
```

---

However, if I rename `gulpfile.js` to `gulpfile.babel.js` and install `babel`, it works fine.
 
**Is that the desired behavior Or should `stylelint` require babel as its own dependency** :question:

**Comment by MoOx** [comment]
node 0.10 is not really supported. Can you try on node 0.12 or iojs 2.x ?

**Comment by MoOx** [comment]
babel is not required and should never be for the distributed version. If to provide a better support, we might just provide some polyfills (maybe via babel/polyfills).

**Comment by sarbbottam** [comment]
Works fine with node 0.12.X.

**Comment by MoOx** [comment]
So we can close?

**Comment by sarbbottam** [comment]
Sure. Thanks.
